tools/matc/src/matc/Config.h

```diff
@@ -41,6 +41,10 @@ class Config {
     using TargetApi = filamat::MaterialBuilder::TargetApi;
     using Optimization = filamat::MaterialBuilder::Optimization;
 
+    // For defines and template args, we use an ordered map with a transparent comparator
+    // to enable efficient lookups using string_view keys.
+    using StringViewMap = std::map<std::string, std::string, std::less<>>;
```

Comment:
I'm a bit confused by the name/comment since we're not using `std::string_view` as the key. Maybe rename to `StringSubstitutionMap` or something?

Reply:
Good point, I renamed the type and rewrote the comment.